.github/workflows/cd.yaml

```
@@ -0,0 +1,82 @@
+        git checkout fork_by_oc  
+        # install ffmpeg 5.1.6
+        sudo scripts/build_ffmpeg.sh nasm yasm x264 x265 opus
+        ./build.sh
+        export BMF_ROOT_PATH=/bmf/output/bmf
+
+    - name: Set up Qt
+      run: |
+        sudo apt-get install -y qt5-qmake qtbase5-dev qtchooser qtbase5-dev-tools cmake build-essential
+
+    - name: Build with CMake
+      run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2046:warning:3:8: Quote this to prevent word splitting

(shellcheck)